tests/unit/client_test.ts

```diff
@@ -75,33 +80,49 @@ for (const browserItem of browserList) {
           await promise;
           await browser.close();
         },
-      );
+        ignore: remote, //ignoring for remote until we figure out how to start a server on the remote container
+      });
     });
 
     await t.step(`close()`, async (t) => {
       await t.step(`Should close all resources and not leak any`, async () => {
-        const { browser, page } = await buildFor(browserItem.name);
+        remote && await waiter();
+        const { browser, page } = await buildFor(browserItem.name, { remote });
         await page.location("https://drash.land");
         await browser.close();
         // If resources are not closed or pending ops or leaked, this test will show it when ran
       });
 
       await t.step(`Should close all page specific resources too`, async () => {
-        const { browser, page } = await buildFor(browserItem.name);
+        remote && await waiter();
+        const { browser, page } = await buildFor(browserItem.name, { remote });
+        let jsonObj = await (await fetch(
+          `http://localhost:9292/json/version`,
+        )).json();
+        const browserWsUrl = jsonObj["webSocketDebuggerUrl"];
```

Comment:
this needed?

Reply:
Yes just to test that the actual browser process was closed, (and because we are respawning the process in shell) the new process has a different ws url